src/libmongoc/src/mongoc/mongoc-stream-tls-secure-channel.c

```diff
@@ -974,24 +974,23 @@ mongoc_stream_tls_secure_channel_new (mongoc_stream_t *base_stream,
    schannel_cred.dwFlags = SCH_USE_STRONG_CRYPTO;
 #endif
 
+   /* By default, enable soft failing.
```

Comment:
Is there a way to opt into a "hard failure" here?  Would that be useful?

Reply:
We could, but I’d prefer to wait unless a need arises though. The other TLS implementations already soft-fail and do not have a way to opt into hard fail, so this is at least consistent. We’d have to add an option that is specific to secure channel, and specific to the C driver.